sqlite/migrations/0007_migrate_replications_add_bucket_name.up.sql

```diff
@@ -0,0 +1,45 @@
+-- Removes the "NOT NULL" from `remote_bucket_id` and adds `remote_bucket_name`.
+ALTER TABLE replications RENAME TO _replications_old;
+
+CREATE TABLE replications
+(
+    id                       VARCHAR(16) NOT NULL PRIMARY KEY,
+    org_id                   VARCHAR(16) NOT NULL,
+    name                     TEXT        NOT NULL,
+    description              TEXT,
+    remote_id                VARCHAR(16) NOT NULL,
+    local_bucket_id          VARCHAR(16) NOT NULL,
+    remote_bucket_id         VARCHAR(16),
```

Comment:
Just a thought - should we add a check constraint that one of remote_bucket_id, remote_bucket_name has a value? like https://stackoverflow.com/questions/3438066/check-constraint-on-multiple-columns